versions.props

```diff
@@ -5,12 +5,8 @@ org.assertj:* = 3.20.2
 org.immutables:value = 2.8.8
 org.junit:junit-bom = 5.7.2
 org.mockito:mockito-core = 3.11.2
+com.netflix.nebula:nebula-test = 10.0.0
 
 # Unnecessary once we have lock files
 com.google.code.findbugs:jsr305 = 3.0.2
 com.google.errorprone:error_prone_annotations = 2.8.1
-
-# dependency-upgrader:OFF
```

Comment:
Excavator failed to open because it didn't update nebula-test. Will also fix this in excavator.